osidb/tests/test_snippet.py

```diff
@@ -51,20 +50,6 @@ def test_create_snippet(self, internal_read_groups, internal_write_groups):
         assert snippet.acl_write == internal_write_groups
         assert Snippet.objects.count() == 1
 
-    def test_snippet_with_wrong_acls(
-        self, embargoed_read_groups, embargoed_write_groups
-    ):
-        """
-        Tests that a snippet with non-internal ACLs raises an error.
-        """
-        snippet = get_snippet()
-
-        snippet.acl_read = embargoed_read_groups
-        snippet.acl_write = embargoed_write_groups
-
-        with pytest.raises(ValidationError, match="Snippet must have internal ACLs."):
-            snippet.save()
-
```

Comment:
Instead of removing the test, you could simply check that after save the ACLs are internal

Reply:
Internal ACLs are tested in `test_create_snippet()` and `test_create_flaw_from_snippet()`.